index.html

```diff
@@ -1727,12 +1727,14 @@ <h3>Issuer</h3>
         <dl>
           <dt><var id="defn-issuer">issuer</var></dt>
           <dd>
-The value of the <code>issuer</code> <a>property</a> MUST be either a
-<a>URL</a> or an object containing an <code>id</code> <a>property</a>. It is
-RECOMMENDED that the <a>URL</a> in the <code>issuer</code> or its
-<code>id</code> be one which, if dereferenced, results in a document containing
-machine-readable information about the <a>issuer</a> that can be used to
-<a>verify</a> the information expressed in the <a>credential</a>.
+The value of the <code>issuer</code> <a>property</a> MUST either be a
+<a>URL</a>, or be an object containing an <code>id</code> <a>property</a>
+whose value is a <a>URL</a>; in either case, the <a>URL</a> MUST identify
+the issuer in a globally unambiguous
+way. It is RECOMMENDED that the <a>URL</a> be one which, if dereferenced,
```

Comment:
How should the verifier check that the `id` is globally unambiguous during the [verification algorithm](https://github.com/w3c/vc-data-model/pull/1338#discussion_r1388509458)?

Reply:
Hrm, you're right, it can't. I was trying to make that "MUST" equate to "you have to be able to uniquely identify the issuer". What if we replace it w/ this:

```suggestion
whose value is a <a>URL</a>; in either case, the <a>URL</a> is used to identify
the issuer using a globally unambiguous identifier.
It is RECOMMENDED that the <a>URL</a> be one which, if dereferenced,
```

Reply:
Or:

```suggestion
whose value is a <a>URL</a>; in either case, the <a>URL</a> is assumed to
be a globally unambiguous identifier for the the issuer.
It is RECOMMENDED that the <a>URL</a> be one which, if dereferenced,
```

Or you could keep the tighter constraint on just the issuer, like:

```suggestion
whose value is a <a>URL</a>; in either case, the issuer MUST select this
<a>URL</a> to identify the issuer in a globally unambiguous
way. It is RECOMMENDED that the <a>URL</a> be one which, if dereferenced,
```

Reply:
I like the stronger, more focused, constraint you proposed on the issuer. That has been merged, thank you!